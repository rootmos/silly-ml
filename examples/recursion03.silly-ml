let rec fib n = match n with
| 0 -> 0
| 1 -> 1
| m -> (fib (m - 1)) + (fib (m - 2));;

print_int (fib 7); print_newline ();;
print_int (fib 8); print_newline ();;
print_int (fib 9); print_newline ();;
