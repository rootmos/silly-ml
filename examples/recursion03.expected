13
21
34
